mono_repo/build.yaml

```diff
@@ -0,0 +1,13 @@
+# Read about `build.yaml` at https://pub.dartlang.org/packages/build_config
+# To update generated code, run `pub run build_runner build`
+targets:
+  mono_repo:
+    builders:
+      json_serializable:
+        options:
+          header: |+
+           // Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
2018

Is there real value in adding copyrights to generated code?

Reply:
> 2018

Eh. I was going for no change in the generated output. We can update that later.

> Is there real value in adding copyrights to generated code?

It's checked-in and included in the package, so I think so...